lib/pbench/server/api/resources/endpoint_configure.py

```diff
@@ -48,12 +48,16 @@ def __init__(self, config: PbenchServerConfig, logger: Logger):
         self.logger = logger
         self.uri_prefix = config.rest_uri
         self.commit_id = config.COMMIT_ID
+        self.server_config = config
```

Comment:
This is mildly annoying:  we're pulling some items out of the config and storing them in the class instance, and then we're storing the whole config in the class instance, too.  I think we should do one or the other.  (I'm OK with adding the secret, client, realm, and server URL as additional attributes; I'm also OK with dropping the attributes for URI prefix and commit ID and pulling them from the config when they are needed; it looks like we only reference each attribute exactly once, so we should do it here _or_ there, but not a mix of both.)